ct-app/economic_handler/economic_handler.py

```diff
@@ -149,6 +210,62 @@ async def get_unique_safe_peerId_links(self):
 
         return "unique_peer_safe_links", response
 
+    async def get_database_metrics(self):
+        """
+        This function establishes a connection to the database using the provided
+        connection details, retrieves the latest peer information from the database
+        table, and returns the data as a dictionary.
+        """
+        with DatabaseConnection(
+            envvar("DB_NAME"),
+            envvar("DB_HOST"),
+            envvar("DB_USER"),
+            envvar("DB_PASSWORD"),
+            envvar("DB_PORT", int),
+        ) as db:
+            try:
+                table_name = "raw_data_table"
+                last_added_rows = db.last_added_rows(table_name)
+
+                metrics_dict = {}
+                for row in last_added_rows:
+                    id, peer_id, netw_ids, latency_metrics, timestamp = row
+
```

Comment:
This can be replaced by (I think):
```suggestion
                for id, peer_id, netw_ids, latency_metrics, timestamp in last_added_rows:
```